Paws4Me/Paws4Me/Extensions/UIImageViewExtension.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  UIImageViewExtension.swift
+//  Paws4Me
+//
+//  Created by Kirsty-Lee Walker on 2022/03/11.
+//
+
+import Foundation
+import UIKit
+
+extension UIImageView {
+    func load(imageURL: String) {
```

Comment:
Look at renaming this function what are we loading?

Reply:
I understand, thank you for your suggestion